python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -169,6 +169,29 @@ def __exit__(self, *exc) -> None:
     def resource_defs(self) -> Optional[Mapping[str, "ResourceDefinition"]]:
         return self._resource_defs
 
+    def merge_resources(self, resources_dict: Mapping[str, Any]) -> "SensorEvaluationContext":
```

Comment:
ah great. almost commented on previous PR that this should be the behavior (merge vs replace)